tests/jobs_fixtures/app57-bond-unbond/storage.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity >=0.0.0;
```

Comment:
Why do we need this file?

Reply:
This is a solc abomination.

solc wants you to tell it what solc version you are expecting your source to be compiled with. else it will give you a warning. You can make this shorter with: "pragma solidity 0;"

I've never seen this in any other programming language. What makes this worse is that the semver expression isn't in quotes so the solang lexer has to jump through hoops to make it work.

Reply:
Oops that was not what you meant. Ignore me.

Reply:
this looks like some errant debug from me... will remove